bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -638,6 +638,13 @@ Switch deciding which nutrient model to use in FATES.
 (Only relevant if FATES is on)
 </entry>
 
+<entry id="fates_seeddisp_cadence" type="integer" category="physics"
+        group="clm_inparm" valid_values="0,1,2,3">
+Switch defining the cadence at which seeds are dispersed across
+gridcells.  Setting the switch value to zero turns off dispersal.
+(Only relevant if FATES is on)
```

Comment:
Also add a note that "1" is only for quick testing, but is slow. Is there a scientific need for monthly?

Reply:
Maybe also add a note that turning this feature on, makes the memory for the feature get larger with the number of processors. Which is in the opposite direction of the rest of the code, where adding more processors makes memory go down. Since, it's only one process and amount of memory so won't necessarily swamp out the overall model memory profile.